models/classes/Copier/CopierServiceProvider.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA.
+ *
+ * @author Andrei Shapiro <andrei.shapiro@taotesting.com>
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoItems\model\Copier;
+
+use oat\tao\model\TaoOntology;
+use oat\oatbox\event\EventManager;
+use taoItems_models_classes_ItemsService;
+use oat\tao\model\resources\Service\ClassCopierProxy;
+use oat\tao\model\resources\Service\ClassPropertyCopier;
+use oat\tao\model\resources\Service\RootClassesListService;
+use oat\generis\model\fileReference\FileReferenceSerializer;
+use oat\tao\model\resources\Service\ClassCopier as TaoClassCopier;
```

Comment:
I would rename the class directly instead of using alias to avoid confusion. Following the same idea as here https://github.com/oat-sa/extension-tao-item/pull/577/files#r863569967

```suggestion
use oat\tao\model\resources\Service\CoreClassCopier;
```

Reply:
I think we can keep simple `ClassCopier`/`InstanceCopier`/etc. for core classes and add a prefix for other sections if needed:
* `ClassCopier` - `ItemClassCopier` - `TestClassCopier` - etc.
* `InstanceCopier` - `ItemCopier` - `TestCopier` - etc.

Wdyt?

Reply:
Sounds good :)